src/elements/app-data.html

```diff
@@ -84,14 +136,6 @@
           schedule: {
             type: Array,
             notify: true
-          },
-
-          _speakersRaw: {
```

Comment:
I think it is redundant to initialize them, coz their initialized while retrieving from FB

 <app-indexeddb-mirror
      session="{$ firebase.indexedDbSession $}"
      key="speakers"
      data="{{_firebaseSpeakersRaw}}"
      persisted-data="{{_speakersRaw}}">
    </app-indexeddb-mirror>